src/sparsezoo/utils/onnx.py

```diff
@@ -121,6 +146,14 @@ def validate_onnx(model: Union[str, ModelProto]):
             return
         onnx.checker.check_model(onnx_model)
     except Exception as err:
```

Comment:
Thought: This seems like a very broad except clause, could we shrink it down to a fewer more specific exceptions maybe?

Reply:
Maybe, but I do not think we should work on this now, may be an act of overoptimization.

Reply:
Still not convinced, but giving a green tick